Integrations/CloudShark/CloudShark.py

```diff
@@ -0,0 +1,204 @@
+import demistomock as demisto
+from CommonServerPython import *
+import requests
+import collections
+
+# disable insecure warnings
+requests.packages.urllib3.disable_warnings()
+
+BASE_URL = demisto.params().get('url')
+INSECURE = demisto.params().get('check_certificate')
+PROXY = demisto.params().get('proxy')
+API_KEY = demisto.params().get('apikey')
+
+if not demisto.params().get('proxy', False):
+    del os.environ['HTTP_PROXY']
+    del os.environ['HTTPS_PROXY']
+    del os.environ['http_proxy']
+    del os.environ['https_proxy']
+
+'''HELPER FUNCTIONS'''
+# Allows nested keys to be accesible
+
+
+def makehash():
+    return collections.defaultdict(makehash)
+
+
+def http(method, url_suffix, params=None, data=None, files=None):
+    response = requests.request(
```

Comment:
please add try/catch section here to catch the following errors:
1. catch the exception created when using bad url - return an error to check the url
2. catch the exception or error created when using bad API key - return an error to check the API key.
to return the error use the common function `return_error()`

this will give the user a clear message for the most common setup errors.